doc/users/next_whats_new/ellipse_annotation.rst

```diff
@@ -0,0 +1,15 @@
+``ellipse`` boxstyle option for annotations
+-------------------------------------------
+
+The `ellipse` option for boxstyle can now be used to create annotations
```

Comment:
These are causing:

```
/home/circleci/project/doc/users/next_whats_new/ellipse_annotation.rst:4: WARNING: py:obj reference target not found: ellipse
/home/circleci/project/doc/users/next_whats_new/ellipse_annotation.rst:4: WARNING: py:obj reference target not found: circle
```

Sphinx using single-back tick to identify things that need to be cross-referenced.  I think the correct fix is to change these to ``'ellipse'`` and ``'circle'``.